node-src/lib/getOptions.ts

```diff
@@ -69,6 +69,7 @@ export default function getOptions({
     forceRebuild: undefined,
     junitReport: undefined,
     zip: undefined,
+    skipUpdateCheck: undefined,
```

Comment:
`skipUpdateCheck: flags.skipUpdateCheck` needs to be added in `optionsFromFlags` down below (around line 126) as well.